src/docgen/view/documentation.ts

```diff
@@ -245,9 +256,9 @@ export class Documentation {
     }
   }
 
-  private async languageSpecific(lang: Language, loose: boolean): Promise<{ assembly: reflect.Assembly; transpile: Transpile}> {
+  private async languageSpecific(lang: Language, options: TransliterationOptions): Promise<{ assembly: reflect.Assembly; transpile: Transpile}> {
```

Comment:
I actually don't like reusing `TransliterationOptions` here because it might encourage duplicate handling of default values. 

This method, while it accepts the same arguments, actually uses a different type, because it intentionally forces those arguments to be required.

i.e

```ts
export interface A {
  readonly y: string;
}
```

is a different type than

```ts
export interface A {
  readonly y?: string;
}
```

Reply:
Since the APIs are private, I updated the parameters to use `Required<>` where appropriate to make this more clear